### ISSUE TITLE
Long status_reason returned from `https://api.comfy.org/nodes/<node>/versions` endpoint

### ISSUE THREAD

**Issue by huchenlei** [Comfy-Org/registry-backend#92]
The returned value is too long, which hurts performance. The `status_reason` should be queried by need.

```
[
  {
    "changelog": "",
    "createdAt": "2024-12-09T07:11:58.955542Z",
    "dependencies": [
      "diffusers",
      "accelerate",
      "clip_interrogator\u003E=0.6.0",
      "sentencepiece",
      "lark-parser",
      "onnxruntime",
      "spandrel",
      "opencv-python"
    ],
    "deprecated": false,
    "downloadUrl": "https://storage.googleapis.com/comfy-registry/yolain/comfyui-easy-use/1.2.5/node.tar.gz",
    "id": "767e9be0-f244-47e7-9053-c1da3bcd9e12",
    "status": "NodeVersionStatusFlagged",
    "status_reason": "[{\"error_type\": \"EvalCall\", \"file_name\": \"/py/image.py\", \"line_number\": 833, \"line\": \"model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/image.py\", \"line_number\": 875, \"line\": \"net.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/easyNodes.py\", \"line_number\": 6464, \"line\": \"upscale_model = model_loading.load_state_dict(sd).eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"OSEnvironCall\", \"file_name\": \"/py/easyNodes.py\", \"line_number\": 7509, \"line\": \"system_root = os.environ.get(\\\"SystemRoot\\\")\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"OSEnvironCall\", \"file_name\": \"/py/easyNodes.py\", \"line_number\": 7510, \"line\": \"user_root = os.environ.get(\\\"USERPROFILE\\\")\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"SubprocessCall\", \"file_name\": \"/py/libs/utils.py\", \"line_number\": 65, \"line\": \"import subprocess\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"SubprocessCall\", \"file_name\": \"/py/libs/utils.py\", \"line_number\": 68, \"line\": \"result = subprocess.run([sys.executable, '-s', '-m', 'pip', 'install', package_command], capture_output=True, text=True)\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/libs/translate.py\", \"line_number\": 100, \"line\": \"zh_en_model = AutoModelForSeq2SeqLM.from_pretrained(zh_en_model_path).eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/libs/translate.py\", \"line_number\": 215, \"line\": \"zh_en_model = AutoModelForSeq2SeqLM.from_pretrained(zh_en_model_path).eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/layer_diffuse/model.py\", \"line_number\": 258, \"line\": \"model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dit/pixArt/loader.py\", \"line_number\": 114, \"line\": \"model.diffusion_model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dit/pixArt/models/pixart_controlnet.py\", \"line_number\": 58, \"line\": \"self.base_model = base_model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"HttpCall\", \"file_name\": \"/py/brushnet/unet_2d_condition.py\", \"line_number\": 7, \"line\": \"#     http://www.apache.org/licenses/LICENSE-2.0\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"HttpCall\", \"file_name\": \"/py/brushnet/unet_2d_blocks.py\", \"line_number\": 7, \"line\": \"#     http://www.apache.org/licenses/LICENSE-2.0\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/ipadapter/__init__.py\", \"line_number\": 230, \"line\": \"self.resampler.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"SubprocessCall\", \"file_name\": \"/py/kolors/loader.py\", \"line_number\": 4, \"line\": \"import subprocess\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/kolors/loader.py\", \"line_number\": 207, \"line\": \"self.text_encoder = ChatGLMModel(textmodel_json_config).eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"SubprocessCall\", \"file_name\": \"/py/kolors/loader.py\", \"line_number\": 213, \"line\": \"subprocess.run([sys.executable, \\\"-m\\\", \\\"pip\\\", \\\"install\\\", \\\"cpm_kernels\\\"], check=True)\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/kolors/chatglm/modeling_chatglm.py\", \"line_number\": 1019, \"line\": \"parameters = eval(content)\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/basics.py\", \"line_number\": 17, \"line\": \"\\\"\\\"\\\"Overwrite model.train with this function to make sure train/eval mode\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 52, \"line\": \"\\\"\\\"\\\"Overwrite model.train with this function to make sure train/eval mode\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 71, \"line\": \"self.transformer = self.transformer.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 114, \"line\": \"self.transformer = self.transformer.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 205, \"line\": \"self.model = self.model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 276, \"line\": \"self.model = self.model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/encoders/condition.py\", \"line_number\": 333, \"line\": \"self.model = self.model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/modules/networks/ae_modules.py\", \"line_number\": 1001, \"line\": \"self.pretrained_model = model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/models/ddpm3d.py\", \"line_number\": 440, \"line\": \"self.first_stage_model = model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/models/ddpm3d.py\", \"line_number\": 448, \"line\": \"self.cond_stage_model = model.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"EvalCall\", \"file_name\": \"/py/dynamiCrafter/lvdm/models/ddpm3d.py\", \"line_number\": 687, \"line\": \"self.embedder = embedder.eval()\", \"scanner\": \"scan_python_file\"}, {\"error_type\": \"contains_url_dependency\", \"scanner\": \"yara_scan\", \"meta\": {\"description\": \"Detects custom wheel or URL dependencies\", \"version\": \"1.1\", \"date\": \"01.08.2024\", \"reference\": \"https://old.reddit.com/r/comfyui/comments/1dbls5n/psa_if_youve_used_the_comfyui_llmvision_node_from/\", \"category\": \"C2\", \"observable_refs\": \"url\", \"attack_id1\": \"T1105 Ingress Tool Transfer\", \"attack_id2\": \"T1059 Command and Scripting Interpreter\", \"severity\": \"3\"}, \"matches\": [{\"filepath\": \"/install.bat\", \"strings\": [{\"identifier\": \"$plaintext_url\", \"instances\": [{\"matched_data\": \"https://pypi.tuna.tsinghua.edu.cn/simple\", \"matched_length\": 40, \"offset\": 47, \"line_number\": 18, \"line\": \"        %aki_python_exec% -s -m pip install -i https://pypi.tuna.tsinghua.edu.cn/simple \\\"%%i\\\"\\n\"}, {\"matched_data\": \"://pypi.tuna.tsinghua.edu.cn/simple\", \"matched_length\": 35, \"offset\": 52, \"line_number\": 18, \"line\": \"        %aki_python_exec% -s -m pip install -i https://pypi.tuna.tsinghua.edu.cn/simple \\\"%%i\\\"\\n\"}]}]}, {\"filepath\": \"/install.sh\", \"strings\": [{\"identifier\": \"$plaintext_url\", \"instances\": [{\"matched_data\": \"https://pypi.tuna.tsinghua.edu.cn/simple\", \"matched_length\": 40, \"offset\": 48, \"line_number\": 17, \"line\": \"        \\\"$aki_python_exec\\\" -s -m pip install -i https://pypi.tuna.tsinghua.edu.cn/simple \\\"$line\\\"\\n\"}, {\"matched_data\": \"://pypi.tuna.tsinghua.edu.cn/simple\", \"matched_length\": 35, \"offset\": 53, \"line_number\": 17, \"line\": \"        \\\"$aki_python_exec\\\" -s -m pip install -i https://pypi.tuna.tsinghua.edu.cn/simple \\\"$line\\\"\\n\"}]}]}, {\"filepath\": \"/.github/FUNDING.yml\", \"strings\": [{\"identifier\": \"$plaintext_url\", \"instances\": [{\"matched_data\": \"https://space.bilibili.com/1840885116\", \"matched_length\": 37, \"offset\": 10, \"line_number\": 3, \"line\": \"custom: [\\\"https://space.bilibili.com/1840885116\\\"]\"}, {\"matched_data\": \"://space.bilibili.com/1840885116\", \"matched_length\": 32, \"offset\": 15, \"line_number\": 3, \"line\": \"custom: [\\\"https://space.bilibili.com/1840885116\\\"]\"}]}]}]}, {\"error_type\": \"contains_url_strings\", \"scanner\": \"yara_scan\", \"meta\": {\"description\": \"Detects URLs in code\", \"version\": \"1.2\", \"date\": \"31.07.2024\", \"category\": \"C2\", \"observable_refs\": \"url\", \"attack_id1\": \"T1102 Web Service\", \"severity\": \"2\"}, \"matches\": [{\"filepath\": \"/py/easyNodes.py\", \"strings\": [{\"identifier\": \"$plaintext_protocol\", \"instances\": [{\"matched_data\": \"https://raw.githubusercontent.com/yolain/ComfyUI-Easy-Use/main/resources/portrait_prompt.json\", \"matched_length\": 93, \"offset\": 78, \"line_number\": 371, \"line\": \"            response = urlopen('https://raw.githubusercontent.com/yolain/ComfyUI-Easy-Use/main/resources/portrait_prompt.json')\\n\"}, {\"matched_data\": \"://raw.githubusercontent.com/yolain/ComfyUI-Easy-Use/main/resources/portrait_prompt.json\", \"matched_length\": 88, \"offset\": 83, \"line_number\": 371, \"line\": \"            response = urlopen('https://raw.githubusercontent.com/yolain/ComfyUI-Easy-Use/main/resources/portrait_prompt.json')\\n\"}]}]}, {\"filepath\": \"/py/config.py\", \"strings\": [{\"identifier\": \"$plaintext_protocol\", \"instances\": [{\"matched_data\": \"https://raw.githubusercontent.com/lllyasviel/Fooocus/main/sdxl_styles/samples\", \"matched_length\": 77, \"offset\": 26, \"line_number\": 46, \"line\": \"FOOOCUS_STYLES_SAMPLES = 'https://raw.githubusercontent.com/lllyasviel/Fooocus/main/sdxl_styles/samples/'\\n\"}, {\"matched_data\": \"://raw.githubusercontent.com/lllyasviel/Fooocus/main/sdxl_styles/samples\", \"matched_length\": 72, \"offset\": 31, \"line_number\": 46, \"line\": \"FOOOCUS_STYLES_SAMPLES = 'https://raw.githubusercontent.com/lllyasviel/Fooocus/main/sdxl_styles/samples/'\\n\"}, {\"matched_data\": \"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/fooocus_inpaint_head.pth\", \"matched_length\": 87, \"offset\": 22, \"line_number\": 49, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/fooocus_inpaint_head.pth\\\"\\n\"}, {\"matched_data\": \"://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/fooocus_inpaint_head.pth\", \"matched_length\": 82, \"offset\": 27, \"line_number\": 49, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/fooocus_inpaint_head.pth\\\"\\n\"}, {\"matched_data\": \"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v26.fooocus.patch\", \"matched_length\": 88, \"offset\": 22, \"line_number\": 54, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v26.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v26.fooocus.patch\", \"matched_length\": 83, \"offset\": 27, \"line_number\": 54, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v26.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v25.fooocus.patch\", \"matched_length\": 88, \"offset\": 22, \"line_number\": 57, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v25.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v25.fooocus.patch\", \"matched_length\": 83, \"offset\": 27, \"line_number\": 57, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint_v25.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint.fooocus.patch\", \"matched_length\": 84, \"offset\": 22, \"line_number\": 60, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint.fooocus.patch\", \"matched_length\": 79, \"offset\": 27, \"line_number\": 60, \"line\": \"        \\\"model_url\\\": \\\"https://huggingface.co/lllyasviel/fooocus_inpaint/resolve/main/inpaint.fooocus.patch\\\"\\n\"}, {\"matched_data\": \"https://huggingface.co/Kijai/BrushNet-fp16/resolve/main/brushnet_random_mask_fp16.safetensors\", \"matched_length\": 93, \"offset\": 26, \"line_number\": 66, \"line\": \"            \\\"model_url\\\": \\\"https://huggingface.co/Kijai/BrushNet-fp16/resolve/main/brushnet_random_mask_fp16.safetensors\\\"\\n\"}, {\"matched_data\": \"://huggingface.co/Kijai/BrushNet-fp16/resolve/main/brushnet_random_mask_fp16.safetensors\", \"matched_length\": 88, \"offset\": 31, \"line_number\": 

... more lines omitted
```

**Comment by huchenlei** [comment]
Context: status_reason was added in https://github.com/Comfy-Org/registry-backend/pull/43

**Comment by huchenlei** [comment]
Resolved in #94
